src/mscorlib/src/System/ReadOnlySpan.cs

```diff
@@ -203,4 +203,62 @@ public bool TryCopyTo(Span<T> destination)
             return true;
         }
     }
+
+    public static class ReadOnlySpanExtensions
+    {
+        /// <summary>
+        /// Creates a new readonly span over the portion of the target string.
+        /// </summary>
+        /// <param name="text">The target string.</param>
+        /// <exception cref="System.ArgumentNullException">Thrown when <paramref name="text"/> is a null
+        /// reference (Nothing in Visual Basic).</exception>
+        public static ReadOnlySpan<char> Slice(this string text)
```

Comment:
@KrzysztofCwalina I can not add implicit cast. I get compiler error `User-defined conversion must convert to or from the enclosing type` (if I add implicit cast to ReadOnlySpan of `T` to cast to ROS of `char`)